src/components/Notification/Notification.js

```diff
@@ -664,6 +665,7 @@ class Notification extends Component {
         className={notificationClasses.toast}>
         <NotificationTextDetails
           title={title}
+          aria-label="catcatcat"
```

Comment:
Did you mean to use an existing prop or introduce a prop here?

Reply:
🤦‍♂️ Ugh, my bad. That was from testing. Will remove.